analysis/dataflow/mark.go

```diff
@@ -49,6 +49,17 @@ const (
 	If
 )
 
+// IndexKind qualifies the kind of index of mark. It is either the index on a value returned by a function,
+// or not an actual index.
+type IndexKind int
+
+const (
+	// ReturnedTupleIndex is  index on a tuple returned by a functoin
+	ReturnedTupleIndex IndexKind = iota
+	// NonIndex is a non-index
+	NonIndex
```

Comment:
Shouldn't `NonIndex` be the default value (0)?

Reply:
I don't think I use the default value, but that's a good point, it should be.